### ISSUE TITLE
Document the .schema and .schemaSum fields of the Replicant class

### ISSUE THREAD

**Issue by Lange** [nodecg/docs#21]
Had some questions about these in Discord today, realized they aren't documented at all. Add those docs here: https://github.com/nodecg/docs/blob/master/docs/classes/replicant.md

**Comment by Lange** [comment]
This was the convo, to give an idea of what these docs should say:

![image](https://user-images.githubusercontent.com/873012/76113561-00293100-5faa-11ea-9064-b8a0c21711f7.png)

**Comment by EwanLyon** [comment]
Should more replicant properties/functions be documented too such as `validate()` and `log`?

**Comment by Lange** [comment]
@EwanLyon yeah, we should document the whole lot. Then, if there's any that I think really shouldn't be used, I can go in and deal with those accordingly.
